docs/firststeps/loggingin.md

```diff
@@ -11,106 +11,56 @@
 [eidas-eduid]: https://puhuri.neic.no/user_guides/myaccessid_registration/ -->
 
 [helpdesk]: ../generic/helpdesk.md
+[setup-ssh-key]: ../firststeps/SSH-keys.md
 
+# Logging in (with SSH client)
 
-# How to log in
 
+Connecting to LUMI via the command line is possible from all major OS. Once you
+have completed the steps to [setting up an SSH key pair][setup-ssh-key] and
+everything has synchronized, you can connect using an ssh client:
 
-<!-- ## Connecting -->
-
-Connecting to LUMI via the command line is possible from all major OS. Once you 
-have generated your key pair and setup your public SSH key on your 
-MyAccessID profile, you can connect using an ssh client:
-
-```
-ssh -i<path-to-private-key> <username>@lumi.csc.fi
+```bash
+ssh -i <path-to-private-key> <username>@lumi.csc.fi
```

Comment:
```suggestion
ssh -i<path-to-private-key> <username>@lumi.csc.fi
```

Reply:
I think there should be no space in between the -i and private-key-path

Reply:
(I think once connecting for the first time, I tried adding a space there first, and it didn't work. But I might not remember this totally correctly. What I remember is that it at least worked when I added NO space, i.e. "ssh -ipathtomyfile username@lumi.csc.fi"

Reply:
Are you sure this is necessary? The man page (`man ssh`) suggests that it should work just fine with a space.

Reply:
Ok, in that case... I'm definitely not sure. If it works on your laptop and especially also if the man page says so, it must be fine

Reply:
And I also tested it. It works for me both with and without the space

Reply:
Alright, I'll keep the version with a space. I think it is easier to read.